framework/doc/acronyms.yml

```
@@ -34,6 +34,8 @@ MWR: Method of Mean Weighted Residuals
 MPI: Message Passing Interface
 MOOSE: Multiphysics Object Oriented Simulation Environment
 NE: Nuclear Energy
+NEAMS: Nuclear Engineering Advanced Modeling and Simulation
```

```suggestion
NEAMS: Nuclear Energy Advanced Modeling and Simulation
```

My mistake here - should be "Energy" instead.

---

👍